docs/topics/mta-cli-transform.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-cli-transform_{context}"]
+= Performing transformation using the command line
+
+Transform has two sub commands - `openrewrite` and `rules`.
+
+[source,terminal,subs="attributes+"]
+----
+Transform application source code or {LC_PSN} XML rules
+
+Usage:
+  {mta-cli} transform [flags]
+  {mta-cli} transform [command]
+
+Available Commands:
+  openrewrite Transform application source code using OpenRewrite recipes
+  rules       Convert XML rules to YAML
+
+Flags:
+  -h, --help   help for transform
+
+Global Flags:
+      --log-level uint32   log level (default 4)
+      --no-cleanup         do not cleanup temporary resources
+
+Use "{mta-cli} transform [command] --help" for more information about a command.
+----
+
+.`OpenRewrite`
+
+The `openrewrite` sub command allows running `OpenRewrite` recipes on source code.
+[source,terminal,subs="attributes+"]
+----
+Transform application source code using OpenRewrite recipes
+
+Usage:
+  {mta-cli} transform openrewrite [flags]
+
+Flags:
+  -g, --goal string     target goal (default "dryRun")
+  -h, --help            help for openrewrite
+  -i, --input string    path to application source code directory
+  -l, --list-targets    list all available OpenRewrite recipes
+  -s, --maven-settings string   path to a custom maven settings file to use
+  -t, --target string   target openrewrite recipe to use. Run --list-targets to get a list of packaged recipes.
+
+Global Flags:
+      --log-level uint32   log level (default 4)
+      --no-cleanup         do not cleanup temporary resources
+----
+To run `transform openrewrite` on application source code, run the following command:
+[source,terminal,subs="attributes+"]
+----
+{mta-cli} transform openrewrite --input=<path/to/source/code> --target=<exactly_one_target_from_the_list>
```

Comment:
Can we make this a note or something to call it out more that you can only use a single target?

I don't know if that is against guidelines or anything, but just a thought